opts/config_test.go

```diff
@@ -0,0 +1,93 @@
+package opts
+
+import (
+	"os"
+	"testing"
+
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestConfigOptionsSimple(t *testing.T) {
```

Comment:
I think the tests here could be refactored to use a table driven test instead 🤔

Reply:
Presumably the same with the Secrets tests